### PR TITLE
Make card picker border follow standards

### PR DESCRIPTION
**Pull request by KTibow**
## Proposed change

This makes the border on the "card"s for the card picker use the same variables as anything `outlined`.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml
frontend:
  themes:
    test_theme:
      ha-card-border-width: 10px
      ha-card-border-color: purple
```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [x] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/editor/card-editor/hui-card-picker.ts [hunk 1]
```diff
@@ -408,6 +407,11 @@ export class HuiCardPicker extends LitElement {
           width: 100%;
           height: 100%;
           z-index: 1;
+          border: var(--ha-card-border-width, 1px) solid var(
+            --ha-card-border-color,
+            var(--divider-color)
+          );
+          border-radius: var(--ha-card-border-radius);
```
Comment (bramkragten) [inline src/panels/lovelace/editor/card-editor/hui-card-picker.ts:414]: Duplicate of line 356

Comment (KTibow) [reply src/panels/lovelace/editor/card-editor/hui-card-picker.ts:414]: Nope, this is applying to the overlay, and 356 is to the card.

Comment (bramkragten) [reply src/panels/lovelace/editor/card-editor/hui-card-picker.ts:414]: Why is the border moved from the card to the overlay?

**Comment by bramkragten** [comment]
Have you set up a dev env and properly tested your PR?

File: src/panels/lovelace/editor/card-editor/hui-card-picker.ts (hunk 1 above)
Comment (KTibow) [reply src/panels/lovelace/editor/card-editor/hui-card-picker.ts:414]: Because it gets cropped off and weird otherwise.

**Comment by KTibow** [comment]
@bramkragten I don't have admin on Windows, so I can't enable the Linux VM. Any way I can do this with gitpod.io?

**Comment by bramkragten** [comment]
Maybe, but I have no expertise in that.

**Comment by zsarnett** [comment]
@KTibow have you found a way to properly test your PR?

**Comment by KTibow** [comment]
After a bit of hassle, I got the dev repo running, and it's passing.

**Comment by zsarnett** [comment]
Did you runn the frontend and see how this looks?

**Comment by KTibow** [comment]
Yup.
image
image
image
Let me know if you need anything else.

**Comment by spacegaier** [comment]
Can you post a screenshot of how the UI looks with the default light and dark themes.

**Comment by KTibow** [comment]
It doesn't make a difference, because for it, `ha-card-border-color` isn't set. It's just like the original. Anyway:
image
image

**Review by bramkragten — APPROVED** [review]

**Pull request merged**